src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const populationAll = [...document.querySelectorAll('.population')].map(
+  (item) => +item.textContent.replaceAll(',', ''),
```

Comment:
Ensure that all elements with the class 'population' contain valid numeric values. If any element contains non-numeric text, the '+' operator will convert it to NaN, which will affect the total and average calculations.